sirepo/package_data/static/js/cloudmc.js

```diff
@@ -544,6 +555,28 @@ SIREPO.app.factory('tallyService', function(appState, cloudmcService, utilities,
         appState.autoSave();
     };
 
+    self.updateThresholds = () => {
+        const s = Object.keys(self.cachedSettings);
+        // if none of the data-specific settings changed, do not update the thresholds
+        if (s.every((k) => self.cachedSettings[k] === appState.models.openmcAnimation[k])) {
+            return;
+        }
+        s.forEach((k) => {
+            self.cachedSettings[k] = appState.models.openmcAnimation[k];
+        });
+        const t = appState.models.openmcAnimation.thresholds;
+        t.global = [self.minField, self.maxField];
+        // since tallies are counts, compare the lower threshold to 0
+        // instead of the global min
+        t.val[0] = Math.max(t.val[0], 0);
+        t.val[1] = Math.min(t.val[1], t.global[1]);
+        // if the lower threshold is now too large, reset to 0
+        if (t.val[0] >= t.val[1]) {
+            t.val[0] = 0;
+        }
+    };
```

Comment:
I think anytime a tally or score changes, the threshold should get reset to min/max for that selection. Otherwise selecting a score could unexpectedly show a partial plot.

Fo example, on the Tokamak example, select the absorption score, followed by the flux score without having made any previous threshold changes.
![x](https://github.com/radiasoft/sirepo/assets/6516910/c02ae1bb-c55c-4be8-ab8f-37562a295187)